core/domain/collection_services.py

```diff
@@ -491,6 +491,36 @@ def get_collection_summaries_matching_query(query_string, cursor=None):
     ], search_cursor)
 
 
+def get_displayable_collections_matching_query(query_string,
```

Comment:
Could you move this to ``core.domain.summary_services`` instead and ensure it's consistent with the corresponding function for exploration summaries? The exp summary version just takes a list of exploration IDs, which I think is all that's needed here, too (except collection IDs).